src/migrations/2017-06-30-interaction-step-answer-option.js

```diff
@@ -0,0 +1,36 @@
+import { r } from '../server/models'
+
+// For each answer_option on interaction steps with question:
+// - Move the value to the answer_option field of the child interaction step
+// - Set the parent_interaction_id of the child to the id of the parent
+// - TODO: Drop the answer_options field
+
+// Run this migration on a running dev or prod instance with the following command:
+// ./dev-tools/babel-run-with-env.js ./src/migrations/2017-06-30-interaction-step-answer-option.js
+
+(async function () {
+  console.log('migrating interaction steps to updated schema');
+  try {
+    const parentSteps = await r.db('spoke')
+      .table('interaction_step')
+      .filter((step) => step('answer_options')
+      .ne([]))
```

Comment:
ok, so when ``answer_options`` isn't equal to an empty array, than it is constitutes as ``parentSteps``